### PR TITLE
Add panoptic driving perception demo using YOLOPv2

### PR DESCRIPTION
**Pull request by DiegoFernandezC**
This PR introduces a new demo using the YOLOPv2 model.

### CODE REVIEW

File: demos/yolopv2/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# YOLOPv2 Example
+
+Simplest possible example of tracking generic objects. Based on [YOLOPv2](https://github.com/CAIC-AD/YOLOPv2).
```
Comment (dekked) [inline demos/yolopv2/README.md:3]: Is it the simplest possible example?

Are we really tracking generic objects? In what dataset was YOLOpv2 trained on?

Comment (dekked) [reply demos/yolopv2/README.md:3]: We should mention this model does Panoptic Driving Perception.

**Review by javiber — COMMENTED** [review]
Can we add some comments on our code that points to the original code in the yolopv2 repo?

The motivation is to separate the code we used from their repo from the one e had to add to make the demo work. This will help us in the future to have the lineage of the code, and know what we did vs what is yolopv2 code. 
Of course, this request is only valid if the distinction is clear, if you had to copy a function from them and then modify it then just add a general comment explaining that the code is based on their repo.

**Review by javiber — APPROVED** [review]

File: demos/yolopv2/Dockerfile [hunk 2]
```diff
@@ -12,6 +12,7 @@ COPY requirements.txt /demo/requirements.txt
 
 RUN pip install -r /demo/requirements.txt
 
-RUN pip install git+https://github.com/tryolabs/norfair.git@master#egg=norfair
+# Install PyTorch from URL
```
Comment (dekked) [inline demos/yolopv2/Dockerfile:15]: This comment is not helpful.

Why are we installing this version of PyTorch?

Comment (DiegoFernandezC) [reply demos/yolopv2/Dockerfile]: We had problems with the versions of Torch used. I didn't see that the actions failed. It was solved using the ones from the docker image.

**Pull request merged**